CMakeLists.txt

```diff
@@ -16,12 +16,13 @@ _SETUP_PROJECT_UNINSTALL()
 option(BUILD_TESTING "Build unit tests" ON)
 option(BUILD_MC_RTC_CLIENT "Build mc_rtc client" ON)
 option(BUILD_OPENRTM_SERVER "Build OpenRTM server component" ON)
+option(BUILD_LEGACY_OPENRTM_SERVER "Build OpenRTM server with c++0x" OFF)
 option(BUILD_DUMMY_CLIENT "Build dummy client" OFF)
 option(BUILD_DUMMY_SERVER "Build dummy server" OFF)
 
 add_subdirectory(src)
 
-if(PYTHON_BINDING)
+if(PYTHON_BINDING AND NOT BUILD_LEGACY_OPENRTM_SERVER)
```

Comment:
I think it's better to rely on the user explicitly providing `PYTHON_BINDING=OFF` if needed with `BUILD_LEGACY_OPENRTM_SERVER` rather than assuming

Reply:
Makes sense, I'll do that then.